packages/fxa-settings/src/components/Settings/BentoMenu/index.tsx

```diff
@@ -113,7 +114,10 @@ export const BentoMenu = () => {
         aria-label={bentoMenuTitle}
         aria-expanded={!!isRevealed}
         aria-haspopup="menu"
-        className="rounded p-2 mx-2 border-transparent hover:bg-grey-100 transition-standard desktop:mx-8"
+        className={classNames(
+          'rounded p-2 mx-2 border-transparent hover:bg-grey-100 transition-standard desktop:mx-8',
+          'focus-visible:outline focus-visible:outline-blue-500 focus-visible:outline-2'
```

Comment:
You asked in Slack if it would be best to extract the focus outlines classes into a utility class - while it's not necessary, it could be nice for easier consistency (your choice!). We could then just add on the offset class separately when needed.